index.html

```diff
@@ -65,6 +65,217 @@ <h3><a> Let's connect </a> </h3>
+                    <li class="position-4">Back End Dev</li>
+                    <li class="year-4">2015</li>
+                </ul>
+            </div>
+            <div class="role-project-4">
+                <p>
+                    A daily selection of privately personalized reads;
+                    no accounts or sign-ups required.
+                </p>
+            </div>
+            <div class="skills-project-4">
+                <ul>
+                    <li>HTML</li>
+                    <li>CSS</li>
+                    <li>javascript</li>
+                </ul>
+            </div>
+            <br>
+            <br>
+            <div class="view-project-4">
+                <button type="button"> See Project</button>
+            </div>
+        </div>
+    </section>
+
+    <section class="about">
+        <div class="about-me-page">
+
+            <div class="about-me-title">
+                <h1> About <br> Myself</h1>
+            </div>
+            <br>
+            <br>
+
+            <div class="about-me-p">
+                <p>
+                    Hello I’m a software developer! I can help you build a product ,
+                    feature or website Look through some of my work and experience!
+                    If you like what you see and have a project you need coded,
+                    don’t hestiate to contact me.
+                </p>
+            </div>
+
+            <br>
+
+            <div class="connect">
+                <h3><a> Let's connect </a> </h3>
+            </div>
+            <br>
+            <div class="social">
+                <ul>
+                    <li class="icon"><a href=""><img src="Image/twitter.png" alt="" /></a></li>
+                    <li class="icon"><a href=""><img src="Image/github.png" alt="" /></a></li>
+                    <li class="icon"><a href=""><img src="Image/medium.png" alt="" /></a></li>
+                    <li class="icon"><a href=""><img src="Image/angellist.png" alt="" /></a></li>
+                </ul>
+
+            </div>
+            <br>
+            <br>
+            <div class="resume">
+                <button type="button"> Get my resume </button>
+            </div>
+            <div class="lang">
+                <h1>Languages</h1>
+            </div>
+
+            <div class="my-lang">
+                <ul>
+                    <li>
+                        <img src="Image/js.png" alt="error"> <h2>javascript</h2>
+                    </li>
+
+                    <li>
+                        <img src="Image/html.png" alt="error"> <h2>HTML</h2>
+                    </li>
+
+                    <li>
+                        <img src="Image/css.png" alt="error"> <h2>css</h2>
+                    </li>
+                </ul>
+            </div>
+
+            <div class="frameworks">
+                <h1>Frameworks</h1>
+            </div>
+            <div class="skills">
+                <h1>Skills</h1>
+            </div>
+        </div>
```

Comment:
Figma Design             |  Your Design
:-------------------------:|:-------------------------:
![skillfig](https://user-images.githubusercontent.com/69638013/153730186-a930f81e-caeb-4bfa-8801-0888c59d7788.PNG)  | ![skillz](https://user-images.githubusercontent.com/69638013/153730193-817a52d7-6007-478e-97d8-cd870b788124.PNG)

- [ ] Kindly add padding and border  to each of the li tags

Reply:
- [ ] Add arrow down and right icons to each titles
- [ ] Kindly add border-bottom to Frameworks and skills titles
###### Check screenshot above ⬆️

Reply:
- [ ] Kindly implement all changes requested from the previous review, please tag @prolajumokeoni if you have any questions.